examples/magento-graphcms/next.config.js

```diff
@@ -3,9 +3,26 @@
 require('dotenv').config({ path: `${__dirname}/.env` })
 
 const { withGraphCommerce } = require('@graphcommerce/next-config')
+const cache = require('next-pwa/cache')
+
+// eslint-disable-next-line import/order
 const withPWA = require('next-pwa')({
   dest: 'public',
   disable: process.env.NODE_ENV === 'development',
+  runtimeCaching: [
+    {
+      urlPattern: /\/_next\/data\/.+\/.+\.json$/i,
+      handler: 'NetworkFirst',
+      options: {
+        cacheName: 'next-data',
+        expiration: {
+          maxEntries: 32,
+          maxAgeSeconds: 24 * 60 * 60, // 24 hours
+        },
+      },
+    },
+    ...cache,
+  ],
```

Comment:
All changes made in the examples/magento-graphcms directory create 'churn' for the integrator when we need to make changes to the caching strategy in the future.

Might it be an idea to export a better runtimeCaching from `@graphcommerce/next-config`?